packages/core/src/helpers/styles/helpers.ts

```diff
@@ -21,7 +21,7 @@ export const nodeHasStyle = (node: MitosisNode) => {
 };
 
 export const hasCss = (component: MitosisComponent) => {
-  let hasStyles = false;
+  let hasStyles = !!component.style?.length;
```

Comment:
might be worth returning early if it's already true, to avoid traversing the entire component:

```ts

```
```suggestion
  let hasStyles = !!component.style?.length;
  
  if (hasStyles) {
    return true
  }
```